.nvmrc

```diff
@@ -1 +1 @@
-lts/gallium
+lts/jod
```

Comment:
Note that this means Node v22, whereas our repo is on v18 (though we want to bump it to v20 as soon as we can remove some blockers from that). That shouldn't matter, right?